README.md

```diff
@@ -7,29 +7,26 @@ A minor Emacs mode for running
 [Clojure test
 mode](https://github.com/technomancy/clojure-mode/blob/master/clojure-test-mode.el).
 
+Note that this is designed for classic expectations, and will not work with the
+[clojure.test version](https://github.com/clojure-expectations/clojure-test)
+
 ## Important
 
-Versions 0.0.2 and lower use swank, version 0.0.3 and greater use
-nrepl. Going forward, only nrepl will be supported.
+This package has some major breakage due to changes in the cider API and removal of
+legacy Emacs functions.
+
+It is probably better to look at CIDER's built-in test features. [CIDER has
+several functions to help you run all your
+tests](https://docs.cider.mx/cider/testing/running_tests.html).
```

Comment:
Note: the reason this mode exists at all is because _classic_ Expectations cannot be run via the normal test running stuff in the Clojure modes and/or CIDER.

If you're using the `clojure.test` version of Expectations, this mode is completely irrelevant and should not be used.